test/Test/Fission/Web/Client/Error.hs

```diff
@@ -0,0 +1,53 @@
+module Test.Fission.Web.Client.Error (tests) where
+
+import           Test.Fission.Prelude as Mock
+
+
+import Fission.Web.Client.Error
+import Network.HTTP.Types.Status
```

Comment:
Align these please! If you're not already, installing stylish-haskell and a plugin for VSCode should autoformat your imports.

https://github.com/vigoo/stylish-haskell-vscode
https://github.com/jaspervdj/stylish-haskell